app/components/project-settings/analysis-settings/regulatory-preference/index.ts

```diff
@@ -132,19 +140,19 @@ export default class ProjectSettingsAnalysisSettingsRegulatoryPreferenceComponen
   });
 
   onResetPcidss = task(async () => {
-    await this.resetPreference.perform('pcidss');
+    await waitForPromise(this.resetPreference.perform('pcidss'));
   });
 
   onResetHipaa = task(async () => {
-    await this.resetPreference.perform('hipaa');
+    await waitForPromise(this.resetPreference.perform('hipaa'));
   });
 
   onResetGdpr = task(async () => {
-    await this.resetPreference.perform('gdpr');
+    await waitForPromise(this.resetPreference.perform('gdpr'));
   });
 
   onResetNist = task(async () => {
-    await this.resetPreference.perform('nist');
+    await waitForPromise(this.resetPreference.perform('nist'));
```

Comment:
is this required ? since we are already waiting in `resetPreference `

Reply:
this test is still not fixed - i have removed it for now it does not make a difference